uperf-client

```diff
@@ -207,9 +207,9 @@ duration=$duration \
 wsize=$wsize \
 rsize=$rsize \
 port=$data_port \
-${cmd} 2>&1 >uperf-client-result.txt
+${cmd} > uperf-client-result.txt 2>&1
 uperf_rc=$?
-if [ $uperf_rc -gt 0 ]; then
-    uperf_errors=`grep -i error uperf-client-result.txt`
+uperf_errors=$(grep -i error uperf-client-result.txt)
+if [ ${uperf_rc} -gt 0 -o -z "${uperf_errors}" ]; then
```

Comment:
I don't like -o  just for the record

Reply:
What would you prefer?

Reply:
`$ a=0; unset b; if [[ ${a} -gt 0 || -z "${b}"  ]]; then echo aaaaaa; fi                                                                                                                                           
aaaaaa
`
My point was about making the code more readable. But the alternative || isn't good, so ignore my comment, -o is good, and it needs only one bracket. Unrelated: if `a` does not exist, this will fail syntax:

`$ unset a; b=bbb; if [ ${a} -gt 0 -o -z "${b}" ]; then echo aaaaaa; fi                                                                                                                                            
bash: [: too many arguments`
Sorry or nit picking, -o is good.